src/cmds/fs/uniq.c

```diff
@@ -0,0 +1,185 @@
+/**
+ * @file
+ * @brief Filter  adjacent matching lines from FILE, writing to standard output.
+ *
+ * @date 02.07.19
+ * @author Nastya Nizharadze
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <ctype.h>
+#include <unistd.h>
+#include <errno.h>
+
+static int number_prefix;
+static int dup_only;
+static int nul_end;
+static int all_dup;
+
+static int max_length;
```

Comment:
You shouldn't  use global variables. It is able to pass as an argument?

Reply:
Yes. I've change it.